tests/target/issue-3981.rs

```diff
@@ -0,0 +1,7 @@
+// rustfmt-normalize_doc_attributes: true
+
+/// item
+fn main() {
+    /// stmt
```

Comment:
Should we not change this?

Reply:
`#[doc = "..."]` and `/// ...` are semantically equivalent, and so we can convert from the attributes to the doc comment as long as the parser accepts it. And since the parser accepts both, the answer is yes :+1: